test/jdk/java/awt/Checkbox/CheckboxCheckerScalingTest.java

```diff
@@ -48,6 +48,7 @@ public class CheckboxCheckerScalingTest {
     private static Checkbox checkbox;
     private static BufferedImage imageAfterChecked;
     private static volatile boolean checkmarkFound = false;
+    private static final int TOLERANCE = 10;
```

Comment:
Is this tolerance needed for the test to pass? Would just moving the test to the center be enough for this fix? I agree with Abhishek that there might be false positives, particularly with color profiles where the test should fail but would pass with this change, especially since this is a ui scaling test.

Reply:
Tolerance is required when we are going with screen capture comparison since I see slight change in intensity of colors in captured image, not really sure of what might be the exact cause for it on some machines/some test runs. Also I've seen some tests considers this tolerance while testing. I confirmed it by decoding the pixel color of captured image (Below image). 
Coming to false positives, yes this is the trade off when the test is made automatic. I see no other way to confirm the check mark with captured image. Other option left is to make the test manual one, yet I feel this way is better than manual. I've captured several runs without the fix where the captured area will be just white background and with fix when checkmark is drawn on UI scale of 2.0 it is draw inside the area without any other interference which might have led to false positive.

Reply:
One more thing, how did you arrive to this number as the tolerance? I assume that maybe other tests also set tolerance as 10?

Reply:
Yes, referenced from other test.